code/game/turfs/closed/wall/reinf_walls.dm

```diff
@@ -3,6 +3,10 @@
 	desc = "A huge chunk of reinforced metal used to separate rooms."
 	icon = 'icons/turf/walls/reinforced_wall.dmi'
 	icon_state = "r_wall"
+	turf_integrity = 200
+	max_integrity = 200
+	heat_capacity = 60000
+	thermal_conductivity = 0.0
```

Comment:
wh ydont they have conductivity at all?

Reply:
oh nevermind saw ur comment in the pr

Reply:
because i want to add shielded walls and while i was testing this the incinerator room was getting hot